fileseed.go

```diff
@@ -22,28 +23,32 @@ func NewIndexSeed(dstFile string, srcFile string, index Index) (*FileSeed, error
 		pos:        make(map[ChunkID][]int),
 		index:      index,
 		canReflink: CanClone(dstFile, srcFile),
+		isInvalid:  false,
 	}
 	for i, c := range s.index.Chunks {
 		s.pos[c.ID] = append(s.pos[c.ID], i)
 	}
 	return &s, nil
 }
 
-// LongestMatchWith returns the longest sequence of of chunks anywhere in Source
-// that match b starting at b[0]. If there is no match, it returns nil
-func (s *FileSeed) LongestMatchWith(chunks []IndexChunk) (int, SeedSegment) {
+// LongestMatchWith returns the longest sequence of chunks anywhere in Source
+// that match b starting at b[0]. If there is no match, it returns a length of zero
```

Comment:
Looks like my comments were wrong, talking about `b` instead of `chunks`. Would you mind fixing them while you're in here?